csv.h

```diff
@@ -315,14 +315,13 @@ class LineReader {
       err.set_file_name(file_name);
       throw err;
     }
-    return std::unique_ptr<ByteSourceBase>(
-        new detail::OwningStdIOByteSourceBase(file));
+    return std::make_unique<detail::OwningStdIOByteSourceBase>(file);
```

Comment:
I see there are several other similar constructions in other LineReader's constructors, I think it could be nice to fix them all

Reply:
You're right! I just created a new PR to solve those instances: https://github.com/swift-nav/fast-cpp-csv-parser/pull/11